### PR TITLE
Added AppContextSwitchOverrides switches for .NET Framework 4.7.1

### PR DESCRIPTION
**Pull request by rpetrusha**
# Added AppContextSwitchOverrides switches for .NET Framework 4.7.1

## Summary
This PR includes the following changes:
- It documents the switches supported by the AppContext class that were added in the .NET Framework 4.7.1.
- It changes fenced references to the AppContextSwitchOverrides element to links.
 
Internal review URLS:
- [AppContext class](https://review.docs.microsoft.com/en-us/dotnet/api/system.appcontext?view=netframework-4.7.1&branch=pr-en-us-3809)
- [\<AppContextSwitchOverrides> element](https://review.docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element?branch=pr-en-us-3809)

## Suggested Reviewers

@mairaw @PriyaPurkayastha

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md [hunk 1]
```diff
@@ -86,13 +89,22 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
...
 |`Switch.System.Windows.Input.Stylus.`<br/>`EnablePointerSupport`|Determines whether an optional `WM_POINTER`-based touch/stylus stack is enabled in WPF applications. For more information, see [Mitigation: Pointer-based Touch and Stylus Support](Mitigation:%20Pointer-based%20Touch%20and%20Stylus%20Support.md) | 
 |`Switch.System.Windows.Media.ImageSourceConverter`<br/>`OverrideExceptionWithNullReferenceException`|Controls whether a legacy [NullReferenceException](xref:System.NullReferenceException) is thrown instead of the exception that more specifically indicates the cause of the exception (such as a [DirectoryNotFoundException](xref:System.IO.DirectoryNotFoundException) or a [FileNotFoundException](xref:System.IO.FileNotFoundException). It is intended for use by code that depends on handling the [NullReferenceException](xref:System.NullReferenceException). | .NET Framework 4.7 |
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md]: not related to your change but the first line seems to be missing the .

File: xml/System/AppContext.xml [hunk 2]
```diff
@@ -275,7 +275,7 @@
   
  In addition to setting the value of a switch programmatically, it can also be set:  
   
--   by adding the switch name and value to the `<AppContextSwitchOverrides>` element in the [\<runtime>](~/docs/framework/configure-apps/file-schema/runtime/runtime-element.md) section of an application configuration file. For example, the following defines a switch named `Libraries.FPLibrary.UseExactFloatingPointComparison` whose value is `False`.  
+-   by adding the switch name and value to the [\<AppContextSwitchOverrides>](~/docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) element in the [\<runtime>](~/docs/framework/configure-apps/file-schema/runtime/runtime-element.md) section of an application configuration file. For example, the following defines a switch named `Libraries.FPLibrary.UseExactFloatingPointComparison` whose value is `False`.
```
Comment (mairaw) [inline xml/System/AppContext.xml]: nit: should the sentence start with lowercase?

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments in case you want to address not related to your changes.

**Review by BillWagner — APPROVED** [review]
LGTM, I did not find anything other than the feedback Maira had already identified.

**Comment by rpetrusha** [comment]
Thanks for the reviews, @BillWagner and @mairaw. I've addressed both comments in the latest commit.

**Pull request merged**